vnext/Microsoft.ReactNative/Views/TextInputViewManager.cpp

```diff
@@ -804,6 +805,7 @@ ShadowNode *TextInputViewManager::createShadow() const {
 
 XamlView TextInputViewManager::CreateViewCore(int64_t /*tag*/, const winrt::Microsoft::ReactNative::JSValueObject &) {
   xaml::Controls::TextBox textBox;
+  EnsureUniqueTextFlyoutForXamlIsland(textBox);
```

Comment:
I wonder if this needs to be called in `registerEvents` since if the secureTextEntry 
 prop changes a new TextBox or PasswordBox is created here: https://github.com/microsoft/react-native-windows/blob/main/vnext/Microsoft.ReactNative/Views/TextInputViewManager.cpp#L508-L535

Reply:
Good call out.